cdiff.py

```diff
@@ -491,14 +572,7 @@ def _fit_with_marker_mix(text, base_color, width, pad=False):
                     count += 1
                     text = text[1:]
 
-            if count == width and tag_re.sub('', text):
-                # Was stripped: output fulfil and still has normal char in text
-                out[-1] = COLORS['reset'] + wrap_char
-            elif count < width and pad:
```

Comment:
Arg `pad` is now unused to in `_fit_with_marker_mix()`.

Reply:
Removed